### PR TITLE
Enable i18n with fr(French) for website

### PR DESCRIPTION
**Pull request by mathieu-benoit**
Like done with `es` here: https://github.com/cncf/tag-app-delivery/pull/594, this PR is enabling the website in `fr`.

Here are the 2 pages translated in French to review at this stage:
- `website/config.toml`
- `website/i18n/fr.toml`

The other files are a copy of `website/content/en` into `website/content/fr` as-is, without any French translation for now (this will be done later in future associated PRs). What was also done is removing these 4 symlinks and the associated content was copied manually:
- `website/content/fr/wgs/platforms/glossary/`
- `website/content/fr/wgs/platforms/maturity-model/`
- `website/content/fr/wgs/platforms/whitepaper/`
- `website/content/fr/wgs/operator/whitepaper`

Also, these following are taken into account:
- https://github.com/cncf/tag-app-delivery/commit/428df6d803c773e993eb88c63fb9398871a0f07f
- https://github.com/cncf/tag-app-delivery/pull/605

When this PR will be ready, the goal will be to have the reviews from:
- [x]  @abangser and team for consistency with other entries
- [ ]  @antoine-sncf for the French translation
- [x] @guillermotti for his recent experience with the `es` translation
- [x] The existing French translation group (who translated the CNCF glossary recently)

### CODE REVIEW

**Comment by mathieu-benoit** [comment]
Ready for your review when you have a minute @abangser and @antoine-sncf, thanks!

**Comment by mathieu-benoit** [comment]
Actually back to draft, as I need to remove the symlinks like in there https://github.com/cncf/tag-app-delivery/pull/599. Still working on this.

**Review by guillermotti — APPROVED** [review]
LGTM!

File: netlify.toml [hunk 1]
```diff
@@ -1,2 +1,2 @@
 [build]
-  ignore = "exit 1"
```
Comment (abangser) [inline netlify.toml:2]: This looks like it may have reverted the commit on main? 🤔 I see the commit that introduced this in your branch which seems off.

**Review by abangser — APPROVED** [review]
This looks really good except that you may have pulled in one commit off main during a rebase. Specifically `Try "exit 1" in both netlify.toml files to force all builds`. 

I think it may be ok (hence the approve) but it may make history look a bit funny!

File: website/content/fr/wgs/artifacts/charter/_index.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+title: Artifacts-WG Charter
+url: 'wgs/artifacts/charter/charter.md/'
```
Comment (abangser) [inline website/content/fr/wgs/artifacts/charter/_index.md:3]: Just learned that this URL line causes issues in the Spanish translation. 

Could you look to remove it? I believe there are a few instances of it across the docs. 

See: https://github.com/cncf/tag-app-delivery/commit/428df6d803c773e993eb88c63fb9398871a0f07f

**Review by abangser — CHANGES_REQUESTED** [review]
Sorry, need to block after learning about the URL field causing issues on the Spanish translation!

I think it's a small fix, but want to try and learn from that. 

The test will be if opening up the maturity model shows as a default language of English or not in the language drop down.

File: website/content/fr/wgs/artifacts/charter/_index.md (hunk 2 above)
Comment (mathieu-benoit) [reply website/content/fr/wgs/artifacts/charter/_index.md]: Thanks for catching and flagging this. Now taken into account.

**Review by abangser — APPROVED** [review]
Thanks for the quick fix @mathieu-benoit! Can confirm that the languages do work correctly, though as you can see in #608 I did spot an unassociated bug which may require a rebase depending on when this gets added.

Very happy for this to be merged now though as it has some french native reviews and appears to work on testing 🙌

**Review by angellk — APPROVED** [review]
lgtm

**Pull request merged**